changelog/fix-6380-edit-dispute-order-note

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
```

Comment:
`nit`: To me it's more an `update` instead of a `fix`.

```suggestion
Type: update
```

Reply:
I agree. Thanks for spotting this 👍 . Updated here https://github.com/Automattic/woocommerce-payments/pull/8371/commits/bee41a4917f48adbe44dcd016432511f894cddbc

Reply:
For the record, I disagree. The old wording was confusing, which was a bug. Improving the wording fixes that bug. 

(Full disclosure, I don't like `update`; `fix` and `add` force us to ask _**why** we are making the change_.)